.github/component-owners.yml

```
@@ -0,0 +1,73 @@
+# This file is used to define the owners of the content modules in the OpenTelemetry repository.
+components:
+  content-modules:
+  - "@open-telemetry/docs-maintainers"
```

I don't think that YAML requires these entries to be quoted. Let's drop the quotes here and elsewhere.

```suggestion
  - @open-telemetry/docs-maintainers
```

---

> Plain value cannot start with reserved character @

I get this annotation from vscode

---

You are right. From the YAML spec:

> The “@” (x40, at) and “`” (x60, grave accent) are reserved for **future use**.